src/index.html

```diff
@@ -5,7 +5,7 @@
         <meta charset="UTF-8">
         <meta http-equiv="X-UA-Compatible" content="IE=edge">
         <meta name="viewport" content="width=device-width, initial-scale=1.0">
-        <title>Document</title>
+        <title>typescript</title>
```

Comment:
Title first word should always be capital
```suggestion
        <title>Typescript</title>
```